### PR TITLE
add(mainApp): Add onboard create account pages -FA-34

### PR DESCRIPTION
**Pull request by hyj1204**
## Description
1. Finish all the pages related to creating account before create profile.
- onboard_pin_page
- onboard_create_account_page
- onboard_recovery_seed_page
- onboard_privacy_setting1_page
- onboard_privacy_setting2_page

https://user-images.githubusercontent.com/14248245/169572671-56604bc0-6568-45a3-b933-70ffa5d8801e.mov

2. Fix the error in UPin dot animation when user switch to another digits pin pad.
3. Add init value in UBottomSheetPin, so it can show a default selected value when it is invoked. If it is null, there is no option checked when it is invoked.
image
4. Add UAppbar widget and its in-code docs.
- UAppBar.back: AppBar with a title and back icon
- UAppBar.iconOnly: AppBar has a list of actions without title
- UAppBar.actions: AppBar has a list of actions and a title
5. Update URecoverySeedBox to have a fixed size.
6. Update AppBarTheme in the dark Utheme
image
7. Add UTertiaryButton(TextButton) and its in-code docs. 
image
8. Update UPin to have a fixed size.
9. Add configuration for saving the screenshot to camera roll: Android(AndroidManifest.xml) and iOS(Info.plist)

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [x] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] ✅ Build configuration change

### CODE REVIEW

**Review by stavares843 — COMMENTED** [review]
on core we are no longer using the settings on the sign up flow

File: lib/auth/onboard_create_account_page.dart [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      body: SafeArea(
+        child: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.fromLTRB(16, 0, 16, 8),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch,
+                children: [
+                  const UText(
+                    'We’re going to create an account for you. On the next screen, you’ll see a set of words. Screenshot this or write it down.',
```
Comment (stavares843) [inline lib/auth/onboard_create_account_page.dart:33]: is not possible to have all these texts as strings on a single file?

Comment (hyj1204) [reply lib/auth/onboard_create_account_page.dart:33]: Yes, we can put all the Strings into a single file.

**Comment by lgmarchi** [comment]
> on core we are no longer using the settings on the sign up flow

@stavares843 

Are you talking about these screens?
image

File: ios/Runner/Info.plist [hunk 2]
```diff
@@ -46,6 +46,8 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>To save the screenshot to your camera roll</string>
```
Comment (lgmarchi) [inline ios/Runner/Info.plist:50]: We will need to change this message later, because it will be used also to take an image from gallery.

**Review by lgmarchi — APPROVED** [review]
It seems pretty good. PR Approved.

File: lib/auth/onboard_create_account_page.dart (hunk 1 above)
Comment (stavares843) [reply lib/auth/onboard_create_account_page.dart:33]: yes that would be better that way we can check all the strings on a single file

**Comment by stavares843** [comment]
> on core we are no longer using the settings on the sign up flow

> @stavares843

> Are you talking about these screens?


yes

**Pull request merged**